roles/caddy/tasks/main.yml

```diff
@@ -34,8 +34,7 @@
 
 - name: install caddy
   shell: |
-    mkdir -p /tmp/caddy-{{ caddy_version }}
-    cd /tmp/caddy-{{ caddy_version }}
```

Comment:
I can see we you thankfully removed the renaming of the file to the version, but still this variable could be out of sync with the binary downloaded, so just download to a random tmp.

Reply:
👍